Mixpanel/Mixpanel.m

```diff
@@ -151,6 +168,11 @@ - (instancetype)initWithToken:(NSString *)apiToken
         self.people = [[MixpanelPeople alloc] initWithMixpanel:self];
         [self setUpListeners];
         [self unarchive];
+        
+        if (optOutTracking) {
```

Comment:
If a customer changes the default implementation from one version to the other, the end user will never be opt-in by default